src/SFA.DAS.EmployerAccounts.Web/Extensions/UrlHelperExtensions.cs

```diff
@@ -54,6 +54,14 @@ public static string EmployerIncentivesAction(this UrlHelper helper, string path
             return Action(baseUrl, $"{hashedAccountId}/{path}");
         }
 
+        public static string YourTransfersAction(this UrlHelper helper, string path = "")
```

Comment:
Is this required? Couldn't we just make use of `EmployerFinanceAction` as the endpoint sits within Finance?

Reply:
Yes, I'll re-use the existing `EmployerFinanceAction` action